### ISSUE TITLE
Allow method calls with files to optionally pass in file-like objects.

### ISSUE THREAD

**Issue by keyz182** [SIMBAChain/PyLibSIMBA#16]
https://github.com/SIMBAChain/PyLibSIMBA/blob/72a87d0072d2d18e73c2be96d6d8aba75e6002ff/pylibsimba/simba.py#L268

**Comment by whelks-chance** [comment]
@keyz182 io.BufferedReader ok?

**Comment by keyz182** [comment]
Any reason not to directly accept the dict that requests is expecting and pass that through? Let the user take care of opening the file/generating a memory stream/creating the file-like-object? 

https://docs.python-requests.org/en/latest/api/#requests.request
```
files – (optional) Dictionary of 'name': file-like-objects (or {'name': file-tuple}) for multipart encoding upload. file-tuple can be a 2-tuple ('filename', fileobj), 3-tuple ('filename', fileobj, 'content_type') or a 4-tuple ('filename', fileobj, 'content_type', custom_headers), where 'content-type' is a string defining the content type of the given file and custom_headers a dict-like object containing additional headers to add for the file.
```

**Comment by whelks-chance** [comment]
Ok sure. That's a change in the interface for that method, but I'll bump a major version and flag it in the README

**Comment by whelks-chance** [comment]
Released v0.3.0

```
Submit files from dict of file objects rather than list of file paths.
```
